packages/flutter_migrate/lib/src/compute.dart

```diff
@@ -0,0 +1,1014 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'base/common.dart';
+import 'base/file_system.dart';
+import 'base/logger.dart';
+import 'base/project.dart';
+import 'base/terminal.dart';
+import 'custom_merge.dart';
+import 'environment.dart';
+import 'flutter_project_metadata.dart';
+import 'result.dart';
+import 'utils.dart';
+
+// This defines files and directories that should be skipped regardless
+// of gitignore and config settings. Using `/` as a stand-in for path separator.
+const List<String> _skippedFiles = <String>[
+  'ios/Runner.xcodeproj/project.pbxproj', // Xcode managed configs that may not merge cleanly.
+  'README.md', // changes to this shouldn't be overwritten since is is user owned.
+];
+
+const List<String> _skippedDirectories = <String>[
```

Comment:
Skipped in what context? You have a generic "skipped" and also a "skipped for merge" in the logic, which makes these ambiguous.